src/e3/python/wheel.py

```diff
@@ -54,23 +53,26 @@ def build(cls, source_dir: str, dest_dir: str, python_tag: str = "py3") -> Wheel
         """
         with tempfile.TemporaryDirectory() as build_dir:
             p = Run(
```

Comment:
Does this go through the `wheels` tests ?

Reply:
According to the coverage, it's tested.